demos/music/lv_demo_music_main.c

```diff
@@ -290,15 +293,18 @@ lv_obj_t * _lv_demo_music_main_create(lv_obj_t * parent)
     lv_obj_t * logo = lv_image_create(lv_screen_active());
     lv_image_set_src(logo, &img_lv_demo_music_logo);
     lv_obj_move_foreground(logo);
+    lv_obj_align_to(logo, spectrum_obj, LV_ALIGN_CENTER, 0, 0);
 
+#if LV_DEMO_MUSIC_SQUARE != 0 && LV_DEMO_MUSIC_ROUND != 0
```

Comment:
Please check if you mean `#if LV_DEMO_MUSIC_SQUARE || LV_DEMO_MUSIC_ROUND`

Reply:
It was really incorrect. The title should be created only for non square and non round display.